testdata/scripts/convert_options_openapiv2.txt

```diff
@@ -57,7 +57,7 @@ type Account struct {
 	//         JSONSchema: openapiv2.JSONSchema{Title: "Account detail", Description: "Detail information of account"},
 	// }
 	// +gunk openapiv2.Schema{
-	//         Example: map[string]string{"value": `{"success":true,"error_code":"0","error_message":""}`},
+	//         Example: openapiv2.Any{Value: []byte(`{"success":true,"error_code":"0","error_message":""}`)},
```

Comment:
was this on purpose?

Reply:
yes, upgrading to the last version of `gunk/opt` introduced this change in `Example` definition.